app/migrations/Version20170926234318.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20170926234318 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $table = $schema->getTable($this->prefix.'reports');
```

Comment:
I'm getting `PHP Notice:  Undefined property: Mautic\Migrations\Version20170926234318::$prefix` on this line. I think it's because you use method `up` and `down` instead of `preUp` and `up`. Look at https://github.com/mautic/mautic/blob/staging/app/migrations/Version20170810120452.php for example. Did you generated this migration with `app/console mautic:migrations:generate`?

Reply:
i made mistake. 
I did call doctrine:migrations:generate
Fixed